### PR TITLE
BULK-Updates-2024-06-06

### PR DESCRIPTION
**Pull request by LauraB-CWF**
Removal of trailing commas and structure 360777770.

### CODE REVIEW

File: data/user_habitat_classification.csv [hunk 1]
```diff
@@ -15247,4 +15247,50 @@
 360886968,0,100,KUMR,SK,spawning,t,SN,2023-09-01,Pacific Salmon Foundation,source point snapped to closest stream within 250m
 360887120,20,120,TSAY,SK,spawning,t,SN,2023-09-01,Pacific Salmon Foundation,source point snapped to closest stream within 250m
 360887414,204,304,KITL,SK,spawning,t,SN,2023-09-01,Pacific Salmon Foundation,source point snapped to closest stream within 250m
-360879788,0,100,NASC,SK,spawning,t,SN,2023-09-01,Pacific Salmon Foundation,manually matched
\ No newline at end of file
+360879788,0,100,NASC,SK,spawning,t,SN,2023-09-01,Pacific Salmon Foundation,manually matched360879788,0,100,NASC,SK,spawning,t,SN,2023-09-01,Pacific Salmon Foundation,manually matched
```
Comment (smnorris) [inline data/user_habitat_classification.csv:15250]: This row is duplicated / missing a line break

**Review by smnorris — CHANGES_REQUESTED** [review]
Should be fine once the duplicate row is removed.

**Comment by smnorris** [comment]
One other question - PSF is noted as source for this new CO/SK observed spawning.
Is there any background on this? Just wondering if this is new data or if it something that was missed on previous data loads.

**Comment by LauraB-CWF** [comment]
Hi Simon,

This is data that was missed. I have images of all the locations too if you want to review.

Thank you,

Laura Bernardi
GIS Technician
***@***.***
350, promenade Michael Cowpland Drive, Kanata, ON K2M 2W1
1.877.599.5777 | 613.599.9594
CanadianWildlifeFederation.ca<http://www.canadianwildlifefederation.ca/?src=signature> | Federationcanadiennedelafaune.ca<http://www.federationcanadiennedelafaune.ca/?src=signature>

[seeds]<http://donate.cwf-fcf.org/site/Donation2?df_id=7440&mfc_pref=T&7440.donation=form1&utm_source=signature&utm_medium=referral&utm_campaign=biod>

Let's celebrate Canada's biodiversity by planting pollinator-friendly seeds.
Request your free seeds ><http://donate.cwf-fcf.org/site/Donation2?df_id=7440&mfc_pref=T&7440.donation=form1?utm_source=signature&utm_medium=email&utm_campaign=mmg>
---------------------------------------
Célébrons la biodiversité du Canada en plantant des semences favorables aux pollinisateurs.
Demandez vos semences gratuites ><http://donate.cwf-fcf.org/site/Donation2?df_id=7440&mfc_pref=T&7440.donation=form1?utm_source=signature&utm_medium=referral&utm_campaign=biod&s_locale=fr_CA>

**Comment by smnorris** [comment]
No need, thanks Laura

**Comment by smnorris** [comment]
A couple of changes that would be great for future fixes:

1. Measure precision
There is no need to record measures to 5 decimal places - rounding to the nearest metre is fine (and preferred)

2. More descriptive notes per PR and per row would be great
Per PR is as per my question above, just some context
Per row mostly is fine to date - but for example I can't tell what things like `Upstream - 110.991 m` refers to in notes

**Comment by LauraB-CWF** [comment]
Hi Simon,

I appreciate this feedback and will keep this in find for all updates I make.

Thank you,

Laura Bernardi
GIS Technician
***@***.***
350, promenade Michael Cowpland Drive, Kanata, ON K2M 2W1
1.877.599.5777 | 613.599.9594
CanadianWildlifeFederation.ca<http://www.canadianwildlifefederation.ca/?src=signature> | Federationcanadiennedelafaune.ca<http://www.federationcanadiennedelafaune.ca/?src=signature>

[seeds]<http://donate.cwf-fcf.org/site/Donation2?df_id=7440&mfc_pref=T&7440.donation=form1&utm_source=signature&utm_medium=referral&utm_campaign=biod>

Let's celebrate Canada's biodiversity by planting pollinator-friendly seeds.
Request your free seeds ><http://donate.cwf-fcf.org/site/Donation2?df_id=7440&mfc_pref=T&7440.donation=form1?utm_source=signature&utm_medium=email&utm_campaign=mmg>
---------------------------------------
Célébrons la biodiversité du Canada en plantant des semences favorables aux pollinisateurs.
Demandez vos semences gratuites ><http://donate.cwf-fcf.org/site/Donation2?df_id=7440&mfc_pref=T&7440.donation=form1?utm_source=signature&utm_medium=referral&utm_campaign=biod&s_locale=fr_CA>

**Pull request merged**